build.gradle

```diff
@@ -82,6 +82,22 @@ allprojects {
     }
 }
 
+configure(allprojects
+        - project("arrow-meta")
+        - project("arrow-ank-gradle")
+        - project("arrow-ank")
+        - project("arrow-kio-benchmarks")
+        - project("arrow-fx")
+        - project("arrow-fx-reactor")
```

Comment:
I believe this both fx projects shouldn't be whitelisted as people shall use them. If the check fails here we could fix them in a separate PR if necessary.